### PR TITLE
[KEBAB] add example to create bot

### PR DESCRIPTION
**Pull request by TheOtterlord**
- Add example to create a bot (lots of people ask for help on discord, now we can point them here)
- Update README on bot example (code switched to api key, but readme did not)

### CODE REVIEW

File: kebab/examples/create-bot/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,113 @@
+{
```
Comment (amitojsingh366) [inline kebab/examples/create-bot/package-lock.json:1]: you should probably switch all of the examples to yarn aswell while you're on it

Comment (theotterlord) [reply kebab/examples/create-bot/package-lock.json:1]: ok, I'll give it a go

Comment (theotterlord) [reply kebab/examples/create-bot/package-lock.json:1]: done (hopefully)

File: package.json [hunk 2]
```diff
@@ -4,6 +4,11 @@
   "workspaces": {
     "packages": [
       "kebab",
+      "kebab/examples/bot",
```
Comment (amitojsingh366) [inline package.json]: nooooo, don't add them to workspaces. just make them use yarn outside of workspaces

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (theotterlord) [reply package.json]: it wouldn't let me because they were not in a workspace. I'll see what I can do though

Comment (theotterlord) [reply package.json]: done :)

**Review by amitojsingh366 — APPROVED** [review]
looks good to me!

**Review by dk-raw — APPROVED** [review]

**Comment by amitojsingh366** [comment]
waiting on you @overlisted

**Review by overlisted — CHANGES_REQUESTED** [review]
don't use yarn in the examples

**Comment by TheOtterlord** [comment]
ok, changing back now

**Comment by TheOtterlord** [comment]
@overlisted restored

**Comment by TheOtterlord** [comment]
All green checks make me happy :)

**Review by overlisted — APPROVED** [review]

**Pull request merged**